api/dependencies/database.py

```diff
@@ -3,7 +3,7 @@
 from .config import conf
 from urllib.parse import quote_plus
 
-SQLALCHEMY_DATABASE_URL = f"mysql+pymysql://{conf.db_user}:{quote_plus(conf.db_password)}@{conf.db_host}:{conf.db_port}/{conf.db_name}?charset=utf8mb4"
+SQLALCHEMY_DATABASE_URL = f"mysql+pymysql://{conf.db_user}:{quote_plus(conf.db_password)}@{conf.db_host}:{conf.db_port}/restaurant_ordering?charset=utf8mb4"
```

Comment:
restaurant_ordering should be {conf.db_name} since we will pull the name from conf.db_name which we have as "sandwich_maker_api"